swift/apple/FirezoneKit/Sources/FirezoneKit/Stores/TunnelStore.swift

```diff
@@ -21,13 +21,6 @@ enum TunnelStoreError: Error {
 public struct TunnelProviderKeys {
   static let keyAuthBaseURLString = "authBaseURLString"
   public static let keyConnlibLogFilter = "connlibLogFilter"
-
-  // The following key is not added to the tunnel provider.
-  // The key is defined so that the key can be removed if it exists
-  // in a tunnel provider configuration. A tunnel configuration
-  // created by an earlier version of the app could have this
-  // key, and we use this key definition to remove it.
-  static let keyAccountId = "accountId"
```

Comment:
I thought more about this and I'd prefer to remove it while we still can. The only users are the Firezone team at this point so this edge case is not worth accounting for.